packages/nunjucks_extensions.mjs

```diff
@@ -0,0 +1,15 @@
+export const setupNunjucksExtensions = (env) => {
```

Comment:
This is the change I'm least comfortable with, as it requires all users of the nunjucks templates enable custom filters.

It is only needed for the conditional check in the radios component.

Alternatively, we could choose to fork this one component only (so there is a radios macro just for Jinja)